packages/react-ui-components/src/Frame/frame.js

```diff
@@ -13,6 +14,18 @@ export default class Frame extends PureComponent {
 
     handleReference = ref => {
         this.ref = ref;
+    };
+
+    componentDidMount() {
+        this.componentDidUpdate();
+    }
+
+    // we do not use react's magic to change to a different URL in the iFrame, but do it explicitely (in order to avoid reloads if we are already on the correct page)
+    componentDidUpdate() {
+        const win = ReactDOM.findDOMNode(this).contentWindow; // eslint-disable-line react/no-find-dom-node
```

Comment:
why not use a ref instead of findDOMNode?

Reply:
would be also fine. I can change it to ref.